code/modules/games/cards.dm

```diff
@@ -65,7 +65,11 @@
 	..()
 
 /obj/item/weapon/deck/attack_hand(mob/user as mob)
-	draw_card()
+	var/mob/living/carbon/human/H = user
+	if(istype(src.loc, /obj/item/weapon/storage) || src.loc == H.r_store || src.loc == H.l_store) // so objects can be removed from storage containers or pockets
```

Comment:
I'm fairly sure ``src.loc == H.r_store || src.loc == H.l_store`` is never going to evaluate to true since if an item is in someone's pockets their loc would equal the mob and not the mob's ``l_store``/``r_store``, which are just ref variables to whatever is in their pockets and so would be the cards themselves.

Reply:
Oh, right, so I need to remove the .loc. Got it. Or I could just check if the loc is the mob?